backend/btrixcloud/pages.py

```diff
@@ -94,6 +94,7 @@ async def init_index(self):
                 ("ts", pymongo.ASCENDING),
             ]
         )
+        await self.pages.create_index([("title", "text")])
```

Comment:
There are notes on the Mongo db pages about $text that indicate [these indices can use quite a huge amount of RAM](https://www.mongodb.com/docs/manual/core/indexes/index-types/index-text/#storage-requirements-and-performance-costs) - since we're doing it on a collection with many millions of records, I'd suggest we test this on dev and try to get a sense of whether it'll be a problem for us before merging